### ISSUE TITLE
Download Json Error

### ISSUE THREAD

**Issue by akiersky** [RealFaviconGenerator/realfavicongenerator#252]
Plugin seems to be very robust, with every option imaginable. I can get through the creation just fine, but when it gets back to my wordpress backend, I get this error:

An error occured: Cannot download JSON file at https://realfavicongenerator.net/files/06b2d074125965dc2e0eb4892c3d99ca09e145bd/json_result.json: SSL connection timeout

Both Chrome and Firefox have the issue. On Safari, the process hangs at "Preparing the favicon editor..."

**Comment by phbernard** [comment]
I'm not aware of any issue like this. In particular, I can download https://realfavicongenerator.net/files/06b2d074125965dc2e0eb4892c3d99ca09e145bd/json_result.json (note that this file will be removed in a few hours, so the link may not work when you give it a try). How is your WP site hosted? Maybe your hosting provider prevents your WP site from connecting to another server.

**Comment by akiersky** [comment]
Hi phbernard, Thanks for the quick reply!
I was able to download it via the link as well, is it possible to load that file to my server somehow to manually install? 

I was able to find this thread: https://wordpress.org/support/topic/ssl-connection-timeout-2 that mentions increasing the network connection's timeout setting in the plugin. I couldn't find that attribute in your plugin's code to test. 

I'm not positive about my server's settings, I will check settings there to see if I have a timeout setting to low.
